tools/deb/BUILD

```diff
@@ -0,0 +1,51 @@
+load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar", "pkg_deb")
+
+pkg_tar(
+    name = "envoy-bin",
+    srcs = [
+        "istio-iptables.sh",
+        "istio-start.sh",
+        "//src/envoy/mixer:envoy",
+    ],
+    mode = "0755",
+    package_dir = "/usr/local/bin",
```

Comment:
A deb package installed binaries and scripts should go to `/usr/bin` (managed by package system), not to `/usr/local/bin` (for locally compiled installation).